### PR TITLE
update README for sencha repository usage

### PR DESCRIPTION
**Pull request by mmoll**
@xalopp @ritters @andythedandy @elger @calbrecht please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,28 @@
...
+[INF] Adding remote "mayflower"
+[INF] Remote "mayflower" added
 ```
-3. [Download](http://cdn.sencha.com/ext/gpl/ext-5.1.1-gpl.zip) and extract the ExtJS framework files into the `ext` directory:
-
-```sh
-    $ unzip <path_to>/ext-5.1.1-gpl.zip
-    $ mv ext-5.1.1 ext
+* Embed the packages
+In your app.js adjust the requires array, e.g.:
```
Comment (xalopp) [inline README.md]: A fixed font is required for file and directory names. Putting single backticks around app.js should do the job.

File: README.md [hunk 2]
```diff
@@ -52,6 +44,37 @@ ExtJS Packages by Mayflower
...
+* [Download](http://cdn.sencha.com/ext/gpl/ext-5.1.1-gpl.zip) and extract the ExtJS framework files into the `ext` directory:
+
+```sh
+    $ unzip <path_to>/ext-5.1.1-gpl.zip
+    $ mv ext-5.1.1 ext
+```
+* build the package with Sencha Cmd:
+
+```sh
+    $ (cd packages/mf-grid-filter-form; sencha package build)
```
Comment (xalopp) [inline README.md]: The build step for `mf-grid-celltooltip` is missing.

File: README.md [hunk 3]
```diff
@@ -52,6 +44,37 @@ ExtJS Packages by Mayflower
...
+```sh
+    $ npm install
+```
+* [Download](http://cdn.sencha.com/ext/gpl/ext-5.1.1-gpl.zip) and extract the ExtJS framework files into the `ext` directory:
+
+```sh
+    $ unzip <path_to>/ext-5.1.1-gpl.zip
+    $ mv ext-5.1.1 ext
+```
+* build the package with Sencha Cmd:
```
Comment (xalopp) [inline README.md:72]: currently we have more thand 1 package => packages

Comment (mmoll) [inline README.md:72]: singular is IMHO correct here.

**Comment by mmoll** [comment]
should be all good now

**Comment by xalopp** [comment]
:+1:

**Pull request merged**